gulp/build.js

```diff
@@ -13,29 +13,28 @@ gulp.task('partials', function () {
   return gulp.src([
     path.join(conf.paths.src, '/components/**/*.html')
   ])
-    .pipe($.minifyHtml({
-      empty: true,
-      spare: true,
-      quotes: true,
-      loose: true
-    }))
+    // TODO: No combination of below's options work; all break the templates.
```

Comment:
@ouranos according to @xaun, this is what was breaking the built files. If you have any clue... Not urgent nor crucial though.